src/ocean/text/convert/UnicodeBom.d

```diff
@@ -381,13 +381,13 @@ class BomSniffer
     private static Const!(Info[]) lookup = [
         {Utf8,  Encoding.Unknown,  null,        true,  false, false, Encoding.UTF_8},
         {Utf8,  Encoding.UTF_8N,   null,        true,  false, false, Encoding.UTF_8},
-        {Utf8,  Encoding.UTF_8,    x"efbbbf",   false},
+        {Utf8,  Encoding.UTF_8,    arrayOf!(char)(0xEF, 0xBB, 0xBF),   false},
         {Utf16, Encoding.UTF_16,   null,        true,  false, false, Encoding.UTF_16BE},
-        {Utf16, Encoding.UTF_16BE, x"feff",     false, true, true},
-        {Utf16, Encoding.UTF_16LE, x"fffe",     false, true},
+        {Utf16, Encoding.UTF_16BE, arrayOf!(char)(0xFE, 0xFF), false, true, true},
+        {Utf16, Encoding.UTF_16LE, arrayOf!(char)(0xFF, 0xFE), false, true},
```

Comment:
`std.conv.hexString` [1], it pays to read the changelog :smiley:.

[1] https://dlang.org/changelog/2.079.0.html#hexstrings

Reply:
What about `"\xFF\xFE"`? Not as nice to read as `x"fffe"` but no library required.

Reply:
The implementation of `hexString` [1] is pretty trivial and could be copied to Ocean to avoid dependencies.

[1] https://github.com/dlang/phobos/blob/27a875709dbbb607795a562b4c20bebe3d38f81e/std/conv.d#L6255

Reply:
Yes, but current workaround is even more trivial. It is not like this functionality is used any regularly.